web/src/components/Terminal.tsx

```diff
@@ -0,0 +1,480 @@
+import clsx from 'clsx'
+import { useEffect, useRef, useState } from 'react'
+
+export default () => {
+    return (
+        <div className='w-full flex flex-col bg-gray-900 rounded-2xl overflow-hidden relative'>
+            <Header />
```

Comment:
I would probably just get rid of the Header component and put it directly in this default export, unless you plan on making this header component into something larger. 

And then I would move Content component to its own file but that's just me

Reply:
Okay there are ~5 components in this one file. We should def break that up

Reply:
Great advice. Done! ✅